### PR TITLE
Generated by robot

### PR DESCRIPTION
**Pull request by caesarxuchao**
All 5 PRs generated in the same run of the robot:
https://github.com/kubernetes/client-go/pull/140
https://github.com/kubernetes/apimachinery/pull/10
https://github.com/kubernetes/apiserver/pull/4
https://github.com/kubernetes/kube-aggregator/pull/4
https://github.com/kubernetes/sample-apiserver/pull/2

### CODE REVIEW

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -637,6 +637,438 @@
...
+		{
+			"ImportPath": "k8s.io/client-go/pkg/util",
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/util/parsers",
+			"Rev": "22e5b7f1e57d431d35c56d97ecc61c0c8a2b597f"
+		},
+		{
+			"ImportPath": "k8s.io/client-go/pkg/version",
```
Comment (sttts) [inline Godeps/Godeps.json]: All these k8s.io repos shouldn't be here.

Comment (sttts) [inline Godeps/Godeps.json]: Or is this Godeps.json only, i.e. without the actual vendor/ files? Then it's ok.

Comment (caesarxuchao) [inline Godeps/Godeps.json]: It's Godeps.json only :)

**Comment by deads2k** [comment]
lgtm

**Comment by deads2k** [comment]
merging in preparation for turning on the auto-sync scripts

**Pull request merged**